### PR TITLE
integration: test lease revoke routine with JWT token

### PR DESCRIPTION
**Pull request by gyuho**
https://github.com/coreos/etcd/pull/9698 wasn't really testing
the panic code path when leases are expiry.

ref. https://github.com/coreos/etcd/issues/9695

### CODE REVIEW

File: integration/v3_auth_test.go [hunk 1]
```diff
@@ -109,9 +109,23 @@ func TestV3AuthRevision(t *testing.T) {
...
+	testV3AuthWithLeaseRevokeWithRoot(t, false)
+}
+
+// TestV3AuthWithLeaseRevokeWithRootJWT creates a lease with a JWT-token enabled cluster.
+// And tests if server is able to revoke expiry lease item.
+func TestV3AuthWithLeaseRevokeWithRootJWT(t *testing.T) {
+	testV3AuthWithLeaseRevokeWithRoot(t, true)
+}
+
+func testV3AuthWithLeaseRevokeWithRoot(t *testing.T, jwt bool) {
```
Comment (joelegasse) [inline integration/v3_auth_test.go]: Rather than adding a flag here, perhaps it would be better to just pass in the `ClusterConfig` that should be used to run the test? If we find more edge cases in the future, it would be easier to add new configurations for the scenario.
